sounddevice.py

```diff
@@ -2444,6 +2444,8 @@ def __init__(self, exclusive=False):
         flags = 0x0
         if exclusive:
             flags |= _lib.paWinWasapiExclusive
+        else:
+            flags |= _lib.paWinWasapiAutoConvert
```

Comment:
I don't think this should be always on (in non-exclusive mode).

What about adding a new named argument?

Something like `sd.WasapiSettings(autoconvert=True)`

Reply:
ok